src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+    private Color[] colors = Color.values();
+
+    public Color getRandomColor() {

```

Comment:
Method 'getRandomColor' should be final to prevent subclasses from changing its behavior.